zappa/core.py

```diff
@@ -624,7 +624,7 @@ def splitpath(path):
                 copytree(site_packages_64, temp_package_path, metadata = False, symlinks=False)
 
         if egg_links:
-            self.copy_editable_packages(egg_links, temp_package_path)
+            self.copy_editable_packages(egg_links, temp_package_path, ignore=shutil.ignore_patterns(*excludes))
```

Comment:
Looks like the remaining CI build failures are because `excludes` isn't necessarily defined by this point because it is set within the `if minify:` blocks above. The quick fix would be to set `excludes = ZIP_EXCLUDES + exclude` before this line, but I don't know if it would be better to move setting `excludes` up and out of the nested conditionals into one spot.